packages/bot-web-ui/src/components/toolbox/toolbox-items.jsx

```diff
@@ -547,7 +547,7 @@ export const ToolboxItems = ReactDomServer.renderToStaticMarkup(
                         text
                     </Field>
                     <Statement name='STACK'>
-                        <Block type='text_statement' movable='false'>
+                        <Block type='text_statement' movable='true'>
```

Comment:
If a user uses this block from the toolbox it works fine but if we have a text_statement in strategy or even saved files, it doesn’t work fine so I suggest instead of set movable manually in this file you can handle it inside text_statment onChange function:
if(event.type === Blockly.Events.BLOCK_CREATE){
            this.setMovable(true)
        }

Reply:
Sure, @mehrzad-fs, that's a very good point!